values.yaml

```diff
@@ -7,8 +7,8 @@ replicaCount: 1
 product: orchestrate
 
 image:
-  repository: consensys-docker-pegasys-orchestrate.bintray.io/orchestrate
-  tag: v2.1.1
+  repository: ""
+  tag: ""
```

Comment:
Same here, having a default version is nice as it works even if you don't specify it